### ISSUE TITLE
Installing on MacOS 10.12

### ISSUE THREAD

**Issue by stevenbarich** [Gcenx/WineskinServer#347]
**Environment (please complete the following information):**
 - OS:  10.12
 - Winery version: 
 - Master Wrapper version: was 2.6.2
 - Engine version: WS9Wine3.13
 - Hardware:  MacBook Pro (17-inch, Late 2011)
 - Application/Game Name 

**Describe the bug**

Hello,
This isn't so much a bug as a "how to uninstall" WineSkinServer.  Let me explain.

I wanted to upgrade my original Wineskin Wrapper to use your Unofficial and new version, which I was very excited to discover.  What I did not understand very clearly is that your new version of Wineskin Wrapper is for MacOS 10.13 and newer.  I use MacOS 10.12, which is what I am limited to on my Macbook Pro 17 from 2011.  

Nonetheless, I went and installed your Wineskin Wrapper 2.9.15, and it completely broke my earlier, working Wineskin 2.6.2 Main Wrapper setup.  All my programs will not work now.  And, I cannot seem to find a way to revert back.  Wineskin 2.6.2 installer is not available online, and even when I revert to a backup of Wineskin 2.6.2 and delete your 2.9.1.5 version, I still have a wrapper error somehow, and I cannot run my programs.  

I fear that the only solution is to re-create my programs, using a restored Wineskin 2.6.2, but...I am concerned there are some hidden files now on my system, that will always continue to search for 2.9.1.5, which simply won't work on my OS.

If you have some advice for me, that would be great and immensely helpful.  If you had a backup copy of the Wineskin 2.6.2 installer, that would also be great if you could post it.  Or even some form of Uninstaller for your Wineskin Wrapper?

Thanks.

**To Reproduce**

**Expected behavior**

**Screenshots**

**LastRunWine.log**

**Additional context**

Incorrectly filled out issues will be close without notice!

**Comment by Gcenx** [comment]
This project only supports macOS High Sierra (10.13) and later as explained at the top of the projects README.

Once you’ve restore the contents of `~/Library/Application Support/Wineskin` from a timemachine backup and restore the old copy of `Wineskin Winery` and trashed any wrappers you’d made with this projects it’s gone.

The thing to remember about Wineskin wrappers is there’s mostly self contained app bundles, meaning they’ll continue to run regardless of what wrapper/engine version is used for another Wineskin based “port”

<br>

I don’t have any backups of Wineskin from doh123, I don’t even think there were offline installers for Wineskin-2.6.2, if you happen to have an old wrapper that used Wineskin-2.6.2 that could be used to recreated the Wineskin-2.6.2 master wrapper.

The next problem would be Engines if you don’t already have all of those somewhere as the ones I’m providing here are a mix of the legacy Winehq packages, then multiple multiple generations of custom Engines some of there’s aren’t even compatible with the next planned master wrapper.

**Comment by stevenbarich** [comment]
Thank you very much for responding.  Sorry to respond so late myself.

After using the latest Wineskin Winery and being able to now download recent engines, I was hopeful that Wineskin master wrapper itself would work, as I had read "...for 10.13 and above..." not parsing that directly as "does not work below 10.13."  My bad, obviously.

Your reply was helpful to confirm what I learned after posting here:  one cannot just replace the Wineskin X app, instead, one needs to replace the entire Wineskin folder.  Which I did, which was successful.

Thanks for the info on engines.  Yes, in my experience, one has to test out every engine to see if it is compatible.  Nonetheless, thanks for your work on this project.  When I can finally upgrade to a computer using 10.13 or higher, I'm sure to use your upgrade.
